src/OhMyThreads.jl

```diff
@@ -15,6 +15,13 @@ export chunks, index_chunks
 
 using TaskLocalValues: TaskLocalValues
 const TaskLocalValue = TaskLocalValues.TaskLocalValue
+
+if isdefined(Base, :ScopedValues)
```

Comment:
ScopedValues.jl already has this code, so you don’t need it here

Reply:
Oh wow, I did not notice that!